### ISSUE TITLE
Inline basic docs? Plugins docs?

### ISSUE THREAD

**Issue by seefood** [Bash-it/bash-it#243]
Just saw a PR with lots of nice aliases and functions supporting git operations. It's nice to discover the new functionalities through the pull requests, but is it also documented in a more orderly fashion somewhere? The Wiki is quite empty, and the plugin directory has no readme(s).

Also, when I first ran the installation, I was asked about which plugins and aliases to turn on, but not a himt about what each option brings with it. I sent the repo to a friend and he just enabled everything, which resulted in a conflict of two plugins and an unusable shell.

Bottom line is, are you planning any sort of one-line descriptions for each component and a longer description somewhere as well, or is it intentionally geared towards people who want to dig into each file and figure it out for themselves? :) I prefer some basic starter documentation to nothing at all...

**Comment by pcuinty** [comment]
I think a good compromise would be to add descriptive comments for each function inside of the plugin.bash files.The git.plugin.bash for example has no comments at all and therefore is very difficult to read

**Comment by ravenhall** [comment]
Plugins that conflict should be reported as bugs with as much detail as can be provided.

I know this thread is old, but it is an interesting suggestion, so **bump**  - perhaps plugins should be required to have some sort of minimal documentation?

**Comment by cornfeedhobo** [comment]
@nwinkler I think this can be closed.

**Comment by seefood** [comment]
Woah. I didn't notice I've been using bash-it for that long...

Well, I'm not sure this has been resolved but if it's not gaining any votes, may as well close it.
